.azure/app-cloud-e2e.yml

```diff
@@ -0,0 +1,163 @@
+# Python package
```

Comment:
I don't think we have e2e tests in this release. For any conflicts, I avoid any updates to this one and updated the milestone in the original PR.

Reply:
todo: update milestone in the original PR (for me)

Reply:
@rohitgr7 - Mind updating the milestone for [this](https://github.com/Lightning-AI/lightning/pull/14539/) PR, I don't have much context about the changes - so if you can do it, it will help.

Reply:
The milestone is correct for it. We want it for both app and pl next releases